examples/smart-contract-upgrade/Contract_Version2/src/lib.rs

```diff
@@ -0,0 +1,212 @@
+//! An example contract that can be upgraded. The contract has a function to
+//! upgrade the smart contract instance to a new module and call optionally a
+//! migration function after the upgrade. To use this example, deploy
+//! `Contract_Version1` and then upgrade the smart contract instance to
+//! `Contract_Version2` by invoking the `upgrade` function with the below JSON
+//! inputParameter:
+//!
+//! {"migrate": {"Some": [["migration",""]]},"module":
+//! <ModuleReferenceContract_Version2>}
```

Comment:
```suggestion
//! {"migrate": {"Some": [["migration",""]]},"module":
//! <ModuleReferenceContract_Version2>}
//! ```json
//! {
//!   "migrate": {
//!     "Some": [
//!       [
//!         "migration",
//!         ""
//!       ]
//!     ]
//!   },
//!   "module": "<ModuleReferenceContract_Version2>"
//! }
//! ```
```